FrontEnd/src/tests/CookieModal.test.js

```diff
@@ -0,0 +1,40 @@
+import { render, screen, cleanup } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+
+import CookieMod from '../components/cookieacception/CookieMod';
+
+afterEach(cleanup);
+
+describe('CookieMod component unit tests', () => {
```

Comment:
Could you add test cases that verify behavior? I mean hiding a modal when agree was clicked

Reply:
Thank you for feedback. Done